readme.rst

```diff
@@ -3,63 +3,42 @@ Documentation starter pack
 
-Then go through the following sections to use this starter pack to set up your docs repository.
+Then go through the following sections to use this starter pack to set up your documentation repository.
 
-Set up your documentation repository
-------------------------------------
+Prerequisites
+-------------
 
-You can either create a standalone documentation project based on this repository or include the files from this repository in a dedicated documentation folder in an existing code repository.
+Before you begin, ensure you have the following:
 
-**Note:** We're planning to provide the contents of this repository as an installable package in the future, but currently, you need to copy and update the required files manually.
+* A GitHub repository where you want to host your documentation, cloned to your local machine. The recommended approach is to host the documentation alongside your code in a `docs` folder. But a standalone documentation repository is also an option; in this case, start with an empty repository.
+* Git and Bash installed on your system.
 
-Standalone documentation repository
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Initialise your documentation repository
+---------------------------------------
 
-To create a standalone documentation repository, clone this starter pack
-repository, `update the configuration <#configure-the-documentation>`_, and
-then commit all files to the documentation repository.
+The ``init.sh`` Bash script is used to initialize your repository with the starter pack content. It adds all the files to your repository that are needed to get started with Sphinx documentation.
 
-You don't need to move any files, and you don't need to do any special
-configuration on Read the Docs.
+To use the script, follow these steps:
 
-Here is one way to do this for newly-created fictional docs repository
-``canonical/alpha-docs``:
+* Copy ``init.sh`` to your repository's root directory.
+* Run the script: ``./init.sh``
+* Enter the installation directory when prompted. For standalone repositories, enter ".". For documentation alongside code, enter the folder where your documentation are e.g. "docs".
 
-.. code-block:: none
+This Bash script does the following:
+
+* Clones the starter pack GitHub repository
+* Creates the specified installation directory if necessary
+* Updates working directory paths in workflow files, and updates configuration paths in the ``.readthedocs.yaml`` file
+* Copies and moves contents and ``.github`` files from the starter pack to the installation directory
+* Deletes the cloned repository when it's done
 
-   git clone git@github.com:canonical/sphinx-docs-starter-pack alpha-docs
-   cd alpha-docs
-   rm -rf .git
-   git init
-   git branch -m main
-   UPDATE THE CONFIGURATION AND BUILD THE DOCS
```

Comment:
This step is very important. The user must do this (without errors) before pushing their changes.

Reply:
I agree that we should keep these instructions.

Reply:
That's not the intention of the new workflow. In the new workflow, the user creates a Git repository in the standard way of their choice, either from GitHub or locally. At the end of that, they have an empty repository, linked to GitHub.

Then, once that's done, they copy over the `init.sh` from the starter pack, which takes care of cloning the starter pack content to a temp dir, installing all the necessary files & updating them, and then deleting the temporary clone.

Personally when I was using the starter pack I found the (old) workflow very confusing. We clone one repo, then delete its `.git` folder, then push to another one. 

I find it much more intuitive to say, start with your own repo (standalone or alongside code), and then install the starter pack using the script. In fact, this was my number 1 motivator to even create this PR.

So not only are the above steps no longer useful with the new workflow, they are in fact misleading. They no longer make any sense.